tools/marvin/marvin/deployDataCenter.py

```diff
@@ -790,8 +790,13 @@ def createZones(self, zones):
                     self.createPrimaryStorages(zone.primaryStorages,
                                                zoneId,
                                                )
-                enabled = getattr(zone, 'enabled', 'True')
-                if enabled == 'True' or enabled is None:
+
+                #get the variable from config file whether to enable zone or not
+                #If "enabled" tag is not found in config file, enable the zone otherwise
+                #act as mentioned in the config file
+                enabled = str(getattr(zone, 'enabled', None))
+                enabled = enabled if enabled else 'true'
+                if enabled.lower() == 'true':
```

Comment:
should work but why not make a str('boolean') a real bool? 
  if enabled.lower == true:
    enable = True
  else:
    enable = False
  if enable: